bench/src/bench.rs

```diff
@@ -2,6 +2,10 @@ use std::hint::black_box as bb;
 
 use criterion::Criterion;
 
+const TZ_NEW_YORK: &str = "America/New_York";
+
+const TZ_SHANGHAI: &str = "Asia/Shanghai";
```

Comment:
Assuming you use `Tz::from_str_insensitive`, I don't think you'll need named constants any more.